packages/design-system/src/components/Table/Table.tsx

```diff
@@ -58,27 +121,31 @@ export class Table extends React.PureComponent {
     }
   }
 
-  componentDidMount() {
+  componentDidMount(): void {
     if (this.props.scrollable) {
       window.addEventListener('resize', this.debounceHandleResize);
       this.handleResize();
     }
   }
 
-  componentWillUnmount() {
+  componentWillUnmount(): void {
     if (this.props.scrollable) {
       window.removeEventListener('resize', this.debounceHandleResize);
     }
   }
 
-  handleResize() {
+  captionID: string;
+  container: any;
```

Comment:
[question] could this variable have a type other than `any`. Looking through the code, it looks like it could be `boolean | React.RefObject<HTMLDivElement>`

Reply:
Typing `container` led to an error on line 142. Leaving `any` for now